pom.xml

```diff
@@ -99,6 +99,11 @@
 			<artifactId>java-string-similarity</artifactId>
 			<version>2.0.0</version>
 		</dependency>
+		<dependency>
+			<groupId>com.github.javaparser</groupId>
+			<artifactId>javaparser-core</artifactId>
+			<version>3.24.7</version>
```

Comment:
```suggestion
			<version>3.24.10</version>
```

Reply:
[Implemented part of the new feedback](https://github.com/ls1intum/Ares/pull/248/commits/ead5d75f0814dbd3738c57b058de8759b980e46d): Updated JavaParser version in the pom.xml.